job-server/src/spark.jobserver/io/JobDAOActor.scala

```diff
@@ -0,0 +1,70 @@
+package spark.jobserver.io
+
+import akka.actor.Props
+import com.typesafe.config.Config
+import ooyala.common.akka.InstrumentedActor
+import org.joda.time.DateTime
+import spark.jobserver.io.JobDAOActor.SaveJar
+import spark.jobserver.util.JarUtils
+
+/**
+ * Created by ankits on 4/6/15.
+ */
+
+object JobDAOActor {
+
+  //Requests
```

Comment:
I'd create a sealed trait base class for all job actor requests, similarly for the responses, and perhaps a task one for Save.. etc.